Contentful.AspNetCore/Contentful.AspNetCore.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup>
+    <Description>Official .NET SDK for the Contentful Content Delivery and management API for ASP.NET core.</Description>
+    <PackageId>contentful.aspnetcore</PackageId>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <VersionPrefix>0.1.0.1-alpha</VersionPrefix>
```

Comment:
Yeah, I agree.. I'll drop the last digit as it seems most dotnet packages actually does that now.